clients/erigon/erigon.sh

```diff
@@ -134,7 +134,7 @@ FLAGS="$FLAGS --ws --ws.port=8546"
 FLAGS="$FLAGS --txpool.blobslots=1000 --txpool.totalblobpoollimit=10000"
 
 # Disable performance optimization incompatible with the tests
-FLAGS="$FLAGS --sync.parallel-state-flushing=false"
+# FLAGS="$FLAGS --sync.parallel-state-flushing=false"
```

Comment:
Required for current erigon pectra branch. Can be added back in the future once updated in erigon.